src/SourceBuild/content/eng/unifiedBuildValidation.targets

```diff
@@ -0,0 +1,13 @@
+<Project>
+
+  <Target Name="RunUnifiedBuildValidation"
+          AfterTargets="Build">
+
+    <MSBuild Projects="../test/Microsoft.DotNet.UnifiedBuild.Tests/Microsoft.DotNet.UnifiedBuild.Tests.csproj"
+             Targets="VSTest"
+             ContinueOnError="true"
+             Properties="ExcludeFromDotNetBuild=false;VsTestUseMSBuildOutput=true" />
+
+  </Target>
```

Comment:
Consider moving this target directly into installer.proj given that the extra file isn't necessary anymore.